foxy/distribution.yaml

```diff
@@ -3612,6 +3612,24 @@ repositories:
       url: https://github.com/ros2/rosidl_typesupport_gurumdds.git
       version: foxy
     status: developed
+  rosxbee:
+    doc:
+      type: git
+      url: https://github.com/Sudharsan10/ROSXBee.git
+      version: 0.0.1
+    release:
+      packages:
+      - rosxbeepy
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/Sudharsan10/ROSXBee-release.git
+      version: 0.0.1-2
+    source:
+      test_pull_requests: true
```

Comment:
you need to follow the `1.1` item of the tutorial [here](http://wiki.ros.org/buildfarm/Pull%20request%20testing) for this to work, did you do that?

Reply:
@ivanpauno Added a webhook as mentioned in the tutorial under `1.1`. Thanks for your comments and review.